cypress/integration/internal/sroc-charge-version-flow.spec.js

```diff
@@ -165,11 +165,19 @@ describe('Create SRoC Charge version workflow journey', () => {
           cy.get('#description').type('@ is an unsupported character');
           cy.get('form > .govuk-button').contains('Continue').click();
           checkInlineAndSummaryErrorMessage(
-            'The description must only include letters a-z, hyphens, numbers and be 180 characters or fewer.');
+            'You can only include letters, numbers, hyphens, the and symbol (&) and brackets. The description must be less than 181 characters.');
           cy.reload();
         });
+        describe('user description contains an unusual but supported character', () => {
+          cy.get('#description').type('* is an supported character');
```

Comment:
this type assertion could cover all supported chars ? 

* & (etc) - are supported characters